pym/bob/cmds/jenkins.py

```diff
@@ -1887,6 +1906,8 @@ def doJenkinsSetOptions(recipes, argv):
         config['credentials'] = args.credentials
     if args.authtoken is not None:
         config['authtoken'] = args.authtoken
+    if args.shortdescription is not False:
```

Comment:
The condition is slightly wrong. It should be `if args.shortdescription is not None` because the parameter can be `True` or `False` depending what the user gave or `None` if no option was given. In this case the parameter should stay unchanged.